core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -1197,7 +1197,7 @@ private boolean checkConsistencyInternal(LockedInodePath inodePath) throws Inval
         return ufs.isDirectory(ufsPath);
       } else {
         return ufs.isFile(ufsPath)
-            && ufs.getFileStatus(ufsPath).getContentLength() == inode.asFile().getLength();
+            && ufs.getExistingFileStatus(ufsPath).getContentLength() == inode.asFile().getLength();
```

Comment:
Changed to `getFileStatus` because if the `ufs.isFile` (a list operation) can find the file, we should be able to get file status (another list operation).